lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -107,6 +107,24 @@ class StorageEnclosureConfig(Command):
             'default': None,
             'optional': True,
             'help': 'Port of the controller to connect to'
+        },
+        'cvg': {
+            'type': int,
+            'default': None,
+            'optional': True,
+            'help': 'cvg number'
+        },
+        'data-devices': {
```

Comment:
Params must be either hypen-separated or underscored. not a mix of both. 
`controller_type` and `data-devices`

Reply:
it was a typo, addressed.